tools/solidity-coverage-example/README.md

```diff
@@ -0,0 +1,29 @@
+# Sample Hedera/Hardhat/Code coverage Project
+
+This project demonstrates a basic usage of the code coverage [plugin](https://github.com/sc-forks/solidity-coverage) of HardHat in a Hedera environment. It comes with a sample contract, a test for that contract, and a script that deploys that contract.
+
+## The integration
+
+You can find a npm script that launches a http server to serve the code coverage report.
+But first the reports have to be generated:
+```shell
+// to run the tests and generate the reports
+npm run test
+// serve the reports on http://127.0.0.1:8080 by default
```

Comment:
Why do I need to start a server, just to check the coverage?
Won't a simple `npx hardhat coverage` be enough, just to show the coverage ?

Reply:
The served report has more detailed data for each file.
But that was a good question. I mentioned the stock command in the README as well (with example output)